src/services/analytics/events/wallet.ts

```diff
@@ -24,3 +24,16 @@ export const WALLET_EVENTS = {
     category: WALLET_CATEGORY,
   },
 }
+
+export const MESSAGE_WALLET_EVENTS = {
```

Comment:
Why a separate export?

Reply:
I wanted to separate them. I've removed the separate export though.